app/test_split_integer.py

```diff
@@ -2,20 +2,24 @@
 
 
 def test_sum_of_the_parts_should_be_equal_to_value() -> None:
-    pass
+    assert (sum(split_integer(50, 5)) == 50)
 
 
 def test_should_split_into_equal_parts_when_value_divisible_by_parts() -> None:
-    pass
```

Comment:
It would be enough to leave only an assert statement to not spread it onto multiple lines. The test name is self-descriptive enough. 

Please, apply similar changes to all functions